### ISSUE TITLE
Save method is not being called

### ISSUE THREAD

**Issue by lakshmi1984** [legalthings/angular-onlyoffice#4]
In my app.js method i am calling the save method. using $scope.save..but when i save document in the onlyOffice editor it doesn't call the mehtod i have put alert in that method but its not being called.
Can anyone please help me in that.

**Comment by jasny** [comment]
@moesjarraf Did we patch OnlyOffice to expose the save method?

**Comment by lakshmi1984** [comment]
From only Office official site i found that onSave event is now deprecated and they are using the callbackUrl in editor config for saving..is it possible to use it in angularjs onlyOffice? if yes how?
I have added the callbackUrl in editorConfig in onlyOffice-editor.js file but the call is not going to the url.I have removed the onsave and other code related to save also.

**Comment by jasny** [comment]
@lakshmi1984 No, since Angular is a so called [single page application](https://en.wikipedia.org/wiki/Single-page_application), you can't work with URLs only with JavaScript calls.

You might need to fork and patch the upstream version of OnlyOffice.
